### PR TITLE
Updated Theming Overview page and images

### LINKED ISSUE #1931: [docs] Graphics for theming overview page
### Description

The [theming overview page](https://deploy-preview-1802--red-hat-design-system.netlify.app/theming/) is still using placeholders, and could use some bespoke svg.

### Links

https://deploy-preview-1802--red-hat-design-system.netlify.app/theming/

### Screenshots

image

### Additional context

More prose would be nice as well, but if that would delay closing this issue, I'd happily settle for just the graphics.

### PR DESCRIPTION
**Pull request by markcaron**
## What I did

1. Updated the [Theming Overview page](https://deploy-preview-1938--red-hat-design-system.netlify.app/theming/) with new content and images

## Testing Instructions

1. Check that [the DP](https://deploy-preview-1938--red-hat-design-system.netlify.app/theming/) matches [the design](https://www.figma.com/proto/3liewFtztT3rP4MKCznjVI/Theming?page-id=1%3A88&node-id=1-89&node-type=canvas&viewport=94%2C132%2C1&t=2ITtBYBVgGpBIwcY-1&scaling=scale-down&content-scaling=fixed)

Closes #1931

### CODE REVIEW

**Review by coreyvickery — APPROVED** [review]

File: docs/theming/index.md [hunk 1]
```diff
@@ -17,22 +18,23 @@ order: 1
...
 </script>
 
-Red Hat's Design System comes with a powerful, flexible, and easy-to-use
-theming system. Design system elements come with branded, accessible defaults 
-built-in, so that all you need to do to create quintessentially Red Hat 
-experiences is write some HTML. For those occasions when we want to flex our 
-design muscles, our elements come with powerful theming primitives in the form 
-of **slots**, **design tokens**, and **CSS shadow parts** to enable you to theme 
-a single element, a section, a page, or an entire app.
+## Overview
```
Comment (adamjohnson) [inline docs/theming/index.md:21]: The h1 is also "Overview". I know the design has "Overview" 2x. Is it necessary? YMMV.

**Review by adamjohnson — APPROVED** [review]
Overview, Overview, YMMV, LGTM

File: docs/theming/index.md (hunk 1 above)
Comment (markcaron) [reply docs/theming/index.md:21]: I debated the same. @coreyvickery please advise.

Comment (markcaron) [reply docs/theming/index.md]: Removed.

**Pull request merged**